qutebrowser/browser/greasemonkey.py

```diff
@@ -94,7 +98,7 @@ def code(self):
         """Return the processed JavaScript code of this script.
 
         Adorns the source code with GM_* methods for Greasemonkey
-        compatibility and wraps it in an IFFE to hide it within a
+        compatibility and wraps it in an iife to hide it within a
```

Comment:
Why? :wink:

Reply:
Why I changed it to be spelled properly is because I keep spelling it wrong.
Why I changed it to be lower case is because I was thinking of some style guide I read some time ago (from the BBC maybe) that suggested that if you pronounce an acronym like a bunch of letters (like FBI) then it should be in all caps but if you pronounce it like a word (like Nasa) it should be captalised like a word.

Reply:
Hah, I didn't even notice the typo fix :laughing: I'd still prefer "IIFE" though as that seems to be the common way it's written everywhere.